src/alto-ui/helpers/number.js

```diff
@@ -18,10 +18,10 @@ const DEFAULT_DECIMAL_SEPARATOR = '.';
 export const round = (n, precision) => Math.round(n * 10 ** precision) / 10 ** precision;
 
 export const parse = (value, locale, precision = 0) => {
-  const parsedValue = parseFloat(value);
-
   const decimalSeparator = decimalSeparatorByLocale[locale] || DEFAULT_DECIMAL_SEPARATOR;
 
+  const parsedValue = parseFloat(value);
```

Comment:
Why this modification ?

Reply:
otherwise, it incorrectly formats the value with decimal places

Reply:
can you update you PR then ?

Reply:
yes, I'll change it back